packages/discord.js/typings/index.d.ts

```diff
@@ -558,6 +566,9 @@ export abstract class CommandInteraction<Cached extends CacheType = CacheType> e
   public inGuild(): this is CommandInteraction<'raw' | 'cached'>;
   public inCachedGuild(): this is CommandInteraction<'cached'>;
   public inRawGuild(): this is CommandInteraction<'raw'>;
+  public deferReply(
+    options: InteractionDeferReplyOptions & { withResponse: true },
+  ): Promise<InteractionCallbackResponse>;
   public deferReply(
     options: InteractionDeferReplyOptions & { fetchReply: true },
```

Comment:
Can you add in a test to ensure that setting both `withResponse: true, fetchReply: true` will return the callback response?